Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.3.3
```

Comment:
Thats the environment its going to be based on (just a 2.3.3 ruby devlopment env). We can change it to whatever. I'll update to 2.2.5